packages/features/portfolio/src/containers/portfolio-overview/containers/leverage-vaults-overview.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box, styled, Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import PositionCard from '../components/position-card';
+import { useVaultHoldingsTable } from '@notional-finance/portfolio-feature-shell/hooks';
+
+interface IProps {
+  data: ReturnType<typeof useVaultHoldingsTable>['vaultHoldingsData'];
+}
+
+const LeverageVaultsOverview = ({ data }: IProps) => {
+  return (
+    <Container>
+      <Heading variant="h2">
+        <FormattedMessage
+          defaultMessage="Leverage Vaults"
+          description="leverage vaults"
+        />
+      </Heading>
+
+      <Box>
+        {data.map((item, i) => (
+          <PositionCard
+            header={{
+              tokenSymbol: item.vault.symbol,
+              tokenName: item.vault.label,
+              description: item.vault.caption,
+            }}
+            data={{
+              ['Health Factor']: {
+                value: item.healthFactor.value,
+                textColor: item.healthFactor.textColor,
+              },
+              ...(item.marketAPY
+                ? {
+                    ['Market APY']: {
+                      value: item.marketAPY,
+                    },
+                  }
+                : {}),
+              ['Present Value']:
+                typeof item.presentValue === 'string'
+                  ? {
+                      value: item.presentValue,
+                    }
+                  : {
+                      value: item.presentValue[0]?.isNegative
```

Comment:
Present Value and Total Earnings are missing, you need to get them from the "data" key